src/routes/+layout.js

```diff
@@ -1 +0,0 @@
-export const prerender = true;
```

Comment:
@billdevmaster afaik this file is required to avoid the build failure: https://github.com/neurolien/neurolien/actions/runs/6253573052/job/16979274643

Reply:
resolved